airflow-core/src/airflow/ui/src/i18n/locales/zh_TW/dashboard.json

```
@@ -0,0 +1,39 @@
+{
+  "welcome": "歡迎",
+  "stats": {
+    "stats": "統計",
+    "failedDags": "失敗的 DAGs",
+    "runningDags": "執行中的 DAGs",
+    "activeDags": "活躍的 DAGs",
+    "queuedDags": "排隊中的 DAGs"
+  },
+  "health": {
+    "health": "健康狀態",
+    "metaDatabase": "元資料庫",
+    "scheduler": "排程器",
+    "triggerer": "觸發器",
+    "dagProcessor": "DAG 處理器"
+  },
+  "poolSlots": "資源池配額",
+  "managePools": "管理資源池",
+  "history": "歷史記錄",
+  "dagRuns": "DAG 執行",
```

```suggestion
  "dagRuns": "Dag 執行紀錄",
```

I think dag runs are basically dag execution history 🤔

---

We should change the translation to _history_ if we add 紀錄 here.

---

Hmm.... yep... probably should just keep it as `Dag 執行`. I know it's a noun but somehow feel likes a verb to me